proxy/common/_compat.py

```diff
@@ -1,9 +1,19 @@
-"""Compatibility code for using Proxy.py across various versions of Python.
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
```

Comment:
Any reason to have unnecessary indentation in every docstring?

Reply:
Not really.  I started fixing these in one of the branches, but will need to revisit later again.

Reply:
Tracking here https://github.com/abhinavsingh/proxy.py/issues/776